src/java.desktop/macosx/classes/com/apple/laf/AquaProgressBarUI.java

```diff
@@ -256,7 +256,7 @@ protected Point getStringPlacement(final Graphics g, final String progressString
             y = oldX;
         }
 
-        return new Point(x + Math.round(width / 2 - stringWidth / 2), y + ((height + fontSizer.getAscent() - fontSizer.getLeading() - fontSizer.getDescent()) / 2) - 1);
+        return new Point(x + (width / 2 - stringWidth / 2), y + ((height + fontSizer.getAscent() - fontSizer.getLeading() - fontSizer.getDescent()) / 2) - 1);
```

Comment:
There was the same bug in the BasicProgressBarUI#getStringPlacement, but it seems it was fixed in the opposite way by replacing the "2" ->> "2.0", see https://github.com/openjdk/jdk/commit/338fb4d94a6a66b21ff39fccf541bbfb63251e6c